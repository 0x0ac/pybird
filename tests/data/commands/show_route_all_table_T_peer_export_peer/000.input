0001 BIRD 1.4.5 ready.
1007-204.130.133.0/24   blackhole [static1 2016-07-24] * (200)
1008-   Type: static unicast univ
1012-   BGP.origin: IGP
        BGP.as_path: 63311 63311
        BGP.next_hop: 10.217.130.42
        BGP.local_pref: 100
        BGP.community: (43531,49021)
1007-157.97.97.0/24     blackhole [static1 2016-09-10] * (200)
1008-   Type: static unicast univ
1012-   BGP.origin: IGP
        BGP.as_path: 63311 63311
        BGP.next_hop: 10.217.130.42
        BGP.local_pref: 100
        BGP.community: (43531,49021)
1007-208.200.137.0/24   blackhole [static1 2016-07-24] * (200)
1008-   Type: static unicast univ
1012-   BGP.origin: IGP
        BGP.as_path: 63311 63311
        BGP.next_hop: 10.217.130.42
        BGP.local_pref: 100
        BGP.community: (43531,49021)
0000
