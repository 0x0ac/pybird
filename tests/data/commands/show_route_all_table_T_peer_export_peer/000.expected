[{"origin": "IGP", "prefix": "204.130.133.0/24", "next_hop": "10.217.130.42", "peer": "[static1", "local_pref": "100", "as_path": "63311 63311", "community": "43531:49021"}, {"origin": "IGP", "prefix": "157.97.97.0/24", "next_hop": "10.217.130.42", "peer": "[static1", "local_pref": "100", "as_path": "63311 63311", "community": "43531:49021"}, {"origin": "IGP", "prefix": "208.200.137.0/24", "next_hop": "10.217.130.42", "peer": "[static1", "local_pref": "100", "as_path": "63311 63311", "community": "43531:49021"}]
